app/views/api/v1/import_puppetclasses/show.json.rabl

```diff
@@ -25,3 +28,7 @@ end
 node(:removed_environment, :if => ->(environment) { @changed['obsolete'][environment.name].present? && @changed['obsolete'][environment.name].match(/_destroy_/) }) do |environment|
   environment.name
 end
+
+node(:ignored_environment, :if => ->(environment) { @changed['ignored'][environment.name].present? && @changed['ignored'][environment.name].match(/_ignored_/) }) do |environment|
```

Comment:
These changes should be made to API v2 - either only v2, or both v2 and v1 if you really wish to continue extending it.